katas/content/single_qubit_gates/index.md

```diff
@@ -15,15 +15,15 @@ This kata introduces you to single-qubit gates. Quantum gates are the quantum co
 
 **What you should know to start working on this kata:**
 
-- Basic linear algebra
-- The concept of qubit
+- Basic knowledge of linear algebra
+- The concept of qubit. If you need a refresher, you can check out "The Qubit" kata preceding this one.
```

Comment:
Again, not sure we should include the pointers in such detail. If we point to The Qubit, why we don't point to Linear Algebra in the first bullet point?